ansible/configure/roles/database-import/tasks/main.yml

```diff
@@ -30,13 +38,20 @@
 - name: Create new database
   command: createdb vmdb_production
 
+- name: Create role 'cloudforms'
+  command: createuser cloudforms
```

Comment:
I've tested this against 55, 56 and 57. And they all throw an error "cloudforms role doesn't exist". So I think I might need to add is55-57 to this. 

Although, I can't comprehend how you never faced this error before while using pg_restore with that same dbdump ?